### PR TITLE
Members overview

### PR DESCRIPTION
**Pull request by miljan-kolosek**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -2,18 +2,20 @@ class User < ApplicationRecord
...
 
+  scope :pending, -> { where.not(invitation_token: true) }
+
   def update_admin
   	self.role = 2 if User.count == 0
   end
 
   def create_slug
-  	self.slug = self.full_name.split(" ")[0].to_s + "-" + self.full_name.split(" ")[1]&.first.to_s
+  	#self.slug = self.full_name.split(" ")[0].to_s + "-" + self.full_name.split(" ")[1]&.first.to_s
```
Comment (tamara-kolosek) [inline app/models/user.rb:19]: delete what is not needed

**Review by tamara-kolosek — COMMENTED** [review]
I didnt find a logic for:
Active link in navigation should be highlighted  (helper)

If I missed it, please point me to that part.

File: app/models/user.rb (hunk 1 above)
Comment (miljan-kolosek) [reply app/models/user.rb:19]: I have a problem with creating a slug for invited users beacuse invited users don't have "full_name" variable, only email.
How can I fix that? Do I need to set default full_name or something like that?

**Comment by miljan-kolosek** [comment]
That part : "Active link in navigation should be highlighted" is not done. Do you have some ideas how to do it in easy way?

File: app/models/user.rb (hunk 1 above)
Comment (tamara-kolosek) [reply app/models/user.rb:19]: It depends on that how you imagined this:
* is admin doing a invitation of already existing users which are already registrated, or
* is admin also registrating the user by doing its invitation. 

If first is the case, just think when are you calling this method and what could be the condition for computing the slug

**Comment by tamara-kolosek** [comment]
@miljan-kolosek For higlighting af an active link, google, it shouldnt be that complicated, maybe ths could help https://mirrorcommunications.com/blog/how-to-create-active-links-using-ruby-on-rails

**Review by tamara-kolosek — APPROVED** [review]

**Pull request merged**